From 575f7975213aa51437fcf2e4c8afae75041b01df Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Fri, 17 Mar 2017 18:31:15 +0800
Subject: [PATCH 0867/1345] fix: dts: a3700: enable comphy by default

commit  463a10e0a47b787ede635c3d8bd364cfbeaa4bdc from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Armada 3700 only has one comphy instance, it should be enabled
by default.
It is the comphy driver's responsibility to disable the dedicated
comphy lanes if it is not used by the IO driver.
JIRA ID: A3700-818 SATA does not work in kernel

Change-Id: Idd004e65866a68b8bbbbc8a906292a98dae32039
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37538
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hua Jing <jinghua@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index aaba920..02f3506 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -470,7 +470,6 @@
 				      <0xe0178 0x8>; /* lane2 indirect access */
 				reg-names = "comphy", "serdes", "usb3_gbe1_phy", "indirect";
 				#phy-cells = <2>;
-				status = "disabled";
 			};
 
 			bm: bm@c0000 {
-- 
1.7.9.5

